KalturaWowzaServer/src/main/java/com/kaltura/media_server/modules/AuthenticationModule.java

```diff
@@ -72,12 +73,15 @@ private KalturaLiveEntry onClientConnect(WMSProperties properties, HashMap<Strin
         int partnerId = Integer.parseInt(requestParams.get(Constants.REQUEST_PROPERTY_PARTNER_ID));
         String entryId = requestParams.get(Constants.REQUEST_PROPERTY_ENTRY_ID);
         String token = requestParams.get(Constants.REQUEST_PROPERTY_TOKEN);
-
-        KalturaEntryServerNodeType serverIndex = KalturaEntryServerNodeType.get(requestParams.get(Constants.REQUEST_PROPERTY_SERVER_INDEX));
+        String partnerStr = "P-" + partnerId;
+                KalturaEntryServerNodeType serverIndex = KalturaEntryServerNodeType.get(requestParams.get(Constants.REQUEST_PROPERTY_SERVER_INDEX));
         KalturaLiveEntry liveEntry = KalturaAPI.getKalturaAPI().authenticate(entryId, partnerId, token, serverIndex);
+        Pair<Object, KalturaEntryIdKey> result = KalturaStreamsDataPersistence.addEntry(entryId, Constants.CLIENT_PROPERTY_KALTURA_LIVE_ENTRY, liveEntry, partnerStr);
```

Comment:
you should call only KalturaentryDataPersistence.setProperty(entryId, Constants.CLIENT_PROPERTY_KALTURA_LIVE_ENTRY, liveEntry)

why the partnerStr it doesn't belong to here, if you want to add more properties then you should call another setProperty